src/components/slider/slider.js

```diff
@@ -643,15 +643,15 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
      * @returns {number}
      */
     function positionToPercent(position) {
-      var offset = vertical ? sliderDimensions.top : sliderDimensions.left;
+      var offset = vertical ? (sliderDimensions.top + window.scrollY) : (sliderDimensions.left + window.scrollX);
       var size = vertical ? sliderDimensions.height : sliderDimensions.width;
       var calc = (position - offset) / size;
 
-      if (!vertical && $mdUtil.isRtl(attr)) {
+      if (vertical || (!vertical && $mdUtil.isRtl(attr))) {
```

Comment:
Can you help me understand why this change is needed? Is this just moving some of the logic from line 654 up to this line?

Reply:
@Splaktar Because it was the exactly same logic. `x = 1 - x`. It just 2 condition for this logic. `If the slider is vertical` OR `if the slider is horizontal and RTL` it then inverse the value. So I prefer to have same code in the same `if` block

Reply:
Maybe we could use exclusive or `^`?

```
if (vertical ^ $mdUtil.isRtl(attr))
```

I felt unfamiliar with XOR in js though

Reply:
No, this is better than XOR. Thank you for the explanation.